examples/cifar10_cnn.py

```diff
@@ -76,11 +83,7 @@
 
 if not data_augmentation:
     print('Not using data augmentation.')
-    model.fit(x_train, y_train,
-              batch_size=batch_size,
-              epochs=epochs,
-              validation_data=(x_test, y_test),
-              shuffle=True)
+    datagen = ImageDataGenerator()
```

Comment:
Please revert this change.

Reply:
Reverting this means if data_augmentation is False that the example crashes. Will I revert this change with the top_k_accuracy and make a new commit just fixing the broken example?

Reply:
The example currently works fine with `data_augmentation=False`...

Reply:
When running with `data_augmentation=False`:

> Traceback (most recent call last):
  File "cifar10_cnn.py", line 132, in <module>
    evaluation = model.evaluate_generator(datagen.flow(x_test, y_test,
NameError: name 'datagen' is not defined`

[Instantiation line](https://github.com/fchollet/keras/blob/master/examples/cifar10_cnn.py#L87)
[Broken line](https://github.com/fchollet/keras/blob/master/examples/cifar10_cnn.py#L132)

Reply:
That's a bug indeed, but of a different kind. Test/evaluation shouldn't be run with data augmentation. Instead we should be using the Numpy array data + `predict`/`evaluate`.

Reply:
I removed this incorrect part of the code in the latest commit. Thanks for pointing it out!

Reply:
Cool :) Will close this PR so.